.pipelines/templates/template-job-deploy-azure-env.yml

```diff
@@ -28,20 +28,7 @@ jobs:
       runOnce:
         deploy:
           steps:
-          - task: DownloadBuildArtifacts@0
```

Comment:
Removed code pulls the config from `deploy pipeline`. To do this properly the `${{ parameters.vsoConfigPipelineID }}` is required, which creates potential point of confusion.

Also versioning is not easy to follow.